solr/core/src/java/org/apache/solr/search/SurroundQParserPlugin.java

```diff
@@ -77,6 +106,16 @@ public Query parse() throws SyntaxError {
           this.maxBasicQueries = DEFMAXBASICQUERIES;
         }
       }
+
+      if (ABSOLUTE_MAX_BASIC_QUERIES > 0 && this.maxBasicQueries > ABSOLUTE_MAX_BASIC_QUERIES) {
```

Comment:
How about just setting value, if that is positive. Probably we don't need to log here.

Reply:
log would be per query, per shard ?

Reply:
It was per query on core I believe.

Since now it's a temporary override (the dev should be well aware of it, unlike the proposal before which could be long living), I have removed the logging :)